website/src/data/gallery.json

```diff
@@ -151,5 +151,12 @@
     "description": "Tutorial on building vision-enabled AI agents using AutoGen and llava.",
     "image": "default.png",
     "tags": ["tools", "app"]
+  },
+  { 
+    "title": "Albus - Enterprise Search and Productivity",
+    "link": "https://www.springworks.in/albus/",
```

Comment:
![image](https://github.com/microsoft/autogen/assets/18212928/8a2181b0-3e0b-40c4-a976-c675db012b94)

there's no indication that autogen is being used , so it's impossible to verify this claim or identify the usecase for autogen.

it would be interesting for the audience if there is something clear to share about autogen if this project is highlighted - just my personal opinion 🙏🏻